domain/src/test/java/org/oppia/android/domain/classify/rules/multiplechoiceinput/MultipleChoiceInputEqualsRuleClassifierProviderTest.kt

```diff
@@ -23,9 +23,9 @@ import kotlin.test.fail
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(manifest = Config.NONE)
 class MultipleChoiceInputEqualsRuleClassifierProviderTest {
-  private val NON_NEGATIVE_VALUE_0 = createNonNegativeInt(value = 0)
-  private val NON_NEGATIVE_VALUE_1 = createNonNegativeInt(value = 1)
-  private val STRING_VALUE_2 = createString(value = "test")
+  private val NON_NEGATIVE_VALUE_0 = InteractionObjectTestBuilder.createNonNegativeInt(value = 0)
```

Comment:
Leave an empty line above, same given in the reference.

Reply:
Done